ec3730/Controllers/PingViewController.swift

```diff
@@ -315,6 +315,9 @@ struct PingSwiftUIViewController: View {
     @State var showAlert: Bool = false
     @State var alertMessage: String?
 
+    @State var isActive = false
```

Comment:
This is not needed. 

Also please rebase onto master

Reply:
PingSetListBranch is already rebased with upstream/mater [https://we.tl/t-TouS1Zc62O](url)

Do we need to rebase it with master [https://we.tl/t-4w5bqWrOHO](url)

as shown in screenshots

Reply:
Your links don't work without copy and pasting. Please fix in the future. 

Yes, rebase with master. You've just been merging which is why this PR has 89 commits.

Reply:
Getting conflicts, can you please confirm which modified file needs to be used, master or PingSetListBranch , attached screenshot.[https://we.tl/t-NM6N1bXT6B](url)